ethclient/ethclient.go

```diff
@@ -806,6 +806,16 @@ func (ec *Client) BundlePrice(ctx context.Context) *big.Int {
 	return price
 }
 
+// Bundles returns the bundles in the given block range
+func (ec *Client) Bundles(ctx context.Context, fromBlock, toBlock *rpc.BlockNumber) ([]*types.BundlesItem, error) {
```

Comment:
do you implement the API standard exactly follow the BEP you draft: https://github.com/bnb-chain/BEPs/pull/460

It seems quite different to me.